Automattic/wp-calypso
# Add code path for eCommerce trial on Plans page

#### Proposed Changes

* As part of https://github.com/Automattic/wp-calypso/issues/72102, this PR adds the new code path in the Plans page for when the selected site has the eCommerce trial plan.

#### Testing Instructions

* Create a site with the new eCommerce Trial plan
* Open the My Plan page (`/plans/<site-slug>`)
* Check if the special path was triggered (`div.ecommerce-trial` is added as a wrapper):
![image](https://user-images.githubusercontent.com/3801502/214147367-e4b169e8-9bb3-4aa7-aada-78647ae598e1.png)
* Sites with other plans shouldn't be affected.
